connector/src/main/java/org/geysermc/connector/utils/FileUtils.java

```diff
@@ -50,6 +55,16 @@
         return objectMapper.readValue(src, valueType);
     }
 
+    public static <T> T loadYaml(InputStream src, Class<T> valueType) throws IOException {
+        ObjectMapper objectMapper = new ObjectMapper(new YAMLFactory()).enable(JsonParser.Feature.IGNORE_UNDEFINED).disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+        return objectMapper.readValue(src, valueType);
+    }
+
+    public static <T> T loadJson(InputStream src, Class<T> valueType) throws IOException {
+        ObjectMapper objectMapper = new ObjectMapper(new JsonFactory()).enable(JsonParser.Feature.IGNORE_UNDEFINED).enable(JsonParser.Feature.ALLOW_COMMENTS).disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
```

Comment:
`GeyserConnector` defines a static OBJECT_MAPPER instance; is that possible to be used here?